ci-operator/step-registry/cucushift/agent/vsphere/postinstall/cucushift-agent-vsphere-postinstall-commands.sh

```diff
@@ -107,12 +107,13 @@ fi
 
 oc patch clusterversion version --type json -p '[{"op": "remove", "path": "/spec/channel"}]}]'
 
-until \
-  oc wait --all=true clusteroperator --for='condition=Available=True' >/dev/null && \
-  oc wait --all=true clusteroperator --for='condition=Progressing=False' >/dev/null && \
-  oc wait --all=true clusteroperator --for='condition=Degraded=False' >/dev/null;  do
-    echo "$(date --rfc-3339=seconds) Clusteroperators not yet ready"
-    sleep 1s
+until
```

Comment:
`oc adm wait-for-stable-cluster --minimum-stable-period=5m --timeout=30m` would do the same, printing the progress

Reply:
Thanks! I didn't know that sub-command existed. 
going to make that change as long as @mhanss approves